data/mainnet/tokens.js

```diff
@@ -45,6 +45,141 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['FARM', 'WETH'],
   },
+  aura_auraBAL: {
```

Comment:
Final small thing, which I forgot about earlier. Please add the tag `isNew: true` to the entries added to `tokens.js`. This will mark the vaults as "New" on the home page.

Reply:
Fixed!